src/Sulu/Bundle/MediaBundle/Tests/Unit/SvgInspector/UploadFileSubscriberTest.php

```diff
@@ -67,6 +78,54 @@ public function testOnKernelRequestWithSafeSvg(): void
         $this->addToAssertionCount(1);
     }
 
+    public function testOnKernelRequestWithReplace(): void
+    {
+        $mimeType = 'image/svg+xml';
+        $uploadedFile = new UploadedFile('test.svg', 'test.svg', $mimeType, 0, true);
+        $newUploadedFile = new UploadedFile('test.svg', 'test.svg', $mimeType, 0, true);
+
+        $request = new Request([], [], [], [], ['file' => $uploadedFile]);
+        $event = $this->createRequestEvent($request);
+
+        $this->svgInspector->supports($mimeType)->willReturn(true);
+        $this->svgInspector->inspect($uploadedFile)->willReturn($newUploadedFile);
+
+        $this->subscriber->onKernelRequest($event);
+
+        $this->assertSame($newUploadedFile, $request->files->get('file'));
+    }
+
+    public function testOnKernelRequestWithNestedFiles(): void
+    {
+        $mimeType = 'image/svg+xml';
+        $uploadedFile = new UploadedFile('test.svg', 'test.svg', $mimeType, 0, true);
+
+        $request = new Request([], [], [], [], ['Product' => ['thumbnail' => $uploadedFile]]);
+        $event = $this->createRequestEvent($request);
+
+        $this->svgInspector->supports($mimeType)->willReturn(true);
+        $this->svgInspector->inspect($uploadedFile)->willReturn($uploadedFile);
```

Comment:
As far as I know this this is missing `shouldBeCalled` calls. Because just specifying the return type of the mock doesn't also assert that this function is actually used.